config/schedule.rb

```diff
@@ -112,6 +112,12 @@ def local(time)
   end
 end
 
+every 1.month, at: local("04:15 am"), roles: [:cron] do
+  if Flipper.enabled?(:ethiopia_dhis2_export)
```

Comment:
Please add an entry to `feature-flags.md`.
IK this is a manual chore, maybe this file can be auto-populated by a bot that raises PR/merges to master every night/weekly.

Reply:
Nice catch, will do.